tests/test_mutualauth.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import unittest
+
+import falcon
+from falcon_mutualauth.mutualauth import Authorize
+
+
+class FakeRequest(object):
+
+    def __init__(self, role):
+        self.__role = role
```

Comment:
Per [Pep-8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles):

```
_single_leading_underscore : weak "internal use" indicator. E.g. from M import * does not import objects whose name starts with an underscore.

single_trailing_underscore_ : used by convention to avoid conflicts with Python keyword, e.g.

Tkinter.Toplevel(master, class_='ClassName')
__double_leading_underscore : when naming a class attribute, invokes name mangling (inside class FooBar, __boo becomes _FooBar__boo ; see below).

__double_leading_and_trailing_underscore__ : "magic" objects or attributes that live in user-controlled namespaces. E.g. __init__ , __import__ or __file__ . Never invent such names; only use them as documented.****
```
The convention I've always seen in python libraries is that a single leading underscore implies a private variable or function. :).